opentelemetry-sdk/tests/trace/test_trace.py

```diff
@@ -545,36 +549,50 @@ def test_span_override_start_and_end_time(self):
     def test_ended_span(self):
         """"Events, attributes are not allowed after span is ended"""
 
-        with self.tracer.start_as_current_span("root") as root:
-            # everything should be empty at the beginning
-            self.assertEqual(len(root.attributes), 0)
-            self.assertEqual(len(root.events), 0)
-            self.assertEqual(len(root.links), 0)
+        # Test that end() is called when the context manager is exited
```

Comment:
I think this test is duplicated.
Test that a span is ended by the context manager is done here https://github.com/open-telemetry/opentelemetry-python/blob/ada53ff5120e3554fe055b1c953b0c3087628ef7/opentelemetry-sdk/tests/trace/test_trace.py#L243
Test that `ends()` prints a warning when called twice is handled in this PR.

Reply:
Ok, I see what you mean, updated.